lib/process-services-cloud/src/lib/people/people-cloud.module.ts

```diff
@@ -33,9 +35,9 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
         CoreModule
     ],
     declarations: [PeopleCloudComponent],
-    exports: [
-        PeopleCloudComponent
-    ]
+    providers: [
+        { provide: PEOPLE_SEARCH_SERVICE_TOKEN, useClass: PeopleCloudService }
```

Comment:
Just a suggestion, can we make this PEOPLE_CLOUD_SEARCH_SERVICE_TOKEN?

Reply:
Yup, we can have  PEOPLE_CLOUD_SEARCH_SERVICE_TOKEN . Thanks